components/features/details/Dashboard.vue

```diff
@@ -9,14 +9,16 @@
       />
     </template>
     <feature-details>
-      <template #subtitle> Assets and Liabilities </template>
-      View all of your assets and liabilities and your exposure to each as a
-      percentage of your net worth.
+      <template #subtitle>
+        {{ $t('home.dashboard.details.asset_and_liabilities.title') }}</template
+      >
```

Comment:
nitpick: maybe we can add a space before the closing to avoid the template tag breaking